exporter/README.md

```diff
@@ -1,17 +1,47 @@
 # Setting up exporters
 
-## Installation
+## Installation (RHEL/CENTOS 7)
 
-* Install latest node_exporter package from Crunchy Repository
-* Install latest postgres_exporter package from Crunchy Repository
-* Install latest pgmonitor-pg##-extras package for your major version of PostgreSQL
-* Install latest crunchy-pg_bloat-check package if you need to monitor database bloat
+RPMs are available to Crunchy customers through our access portal. The RPM will take care of all the steps below in the Installation section. From the Setup section onward, the instructions are relevant no matter the installation method.
```

Comment:
There are currently RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the Crunchy Data [access portal](https://access.crunchydata.com/).  Installing the RPM will take care of all of the steps described in the installation section below.  If you install via the RPM, you can continue reading at the [Service Setup](#service-setup) section.